docs/docsite/rst/network/user_guide/faq.rst

```diff
@@ -0,0 +1,34 @@
+.. _network_faq:
+
+*******************
+Ansible Network FAQ
+*******************
+
+How can I improve performance for network playbooks?
```

Comment:
@privateip had a way of avoiding all the skipped tasks (network_os) which may be relevant for this section. He mentioned it during the four playbook exercises in Durham